extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -211,7 +211,7 @@ export class AmpIframe extends AMP.BaseElement {
           this.element);
       return;
     }
-    this.requestChangeHeight(newHeight);
+    this./*REVIEW*/changeHeightIfAllowed(newHeight);
```

Comment:
We shouldn't require `/*REVIEW*/` for `changeHeightIfAllowed` - that's what it's for.